### PR TITLE
Update README with the changes in #179

### PR DESCRIPTION
**Pull request by ruippeixotog**

### CODE REVIEW

File: docs/src/main/tut/configurable-converters.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## Configurable converters
 
-For some types, PureConfig cannot automatically derive a converter because there are multiple ways to convert a configuration value to them. For instance, for [`LocalDate`](https://docs.oracle.com/javase/8/docs/api/java/time/LocalDate.html) PureConfig cannot derive a converter because there are multiple [`DateTimeFormatter`](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html)s that can be used to convert a String into a `LocalDate`. Examples of different formats are `yyyy-mm-dd`, e.g. `"2016-01-01"`, and `yyyymmdd`, e.g. `"20160101"`. 
+For some types, PureConfig cannot automatically derive a converter because there are multiple ways to convert a configuration value to them. For instance, for [`LocalDate`](https://docs.oracle.com/javase/8/docs/api/java/time/LocalDate.html) PureConfig cannot derive a converter because there are multiple [`DateTimeFormatter`](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html)s that can be used to convert a String into a `LocalDate`. Examples of different formats are `yyyy-mm-dd`, e.g. `"2016-01-01"`, and `yyyymmdd`, e.g. `"20160101"`.
```
Comment (derekmorr) [inline docs/src/main/tut/configurable-converters.md:3]: What changed here? I don't see what's different?

**Review by derekmorr — APPROVED** [review]
LGTM.

File: docs/src/main/tut/configurable-converters.md (hunk 1 above)
Comment (ruippeixotog) [reply docs/src/main/tut/configurable-converters.md:3]: I don't really know, I can't see it either... Maybe some whitespace? I guess I opened the file in IntelliJ and it reformatted the file in some way.

File: core/src/main/scala/pureconfig/package.scala [hunk 2]
```diff
@@ -170,8 +170,8 @@ package object pureconfig {
    * @return the configuration
    */
   @throws[ConfigReaderException[_]]
-  def loadConfigOrThrow[Config](path: Path, namespace: String)(implicit conv: ConfigReader[Config], ct: ClassTag[Config]): Config = {
-    getResultOrThrow[Config](loadConfig(path, namespace)(conv))
+  def loadConfigOrThrow[Config](path: Path, namespace: String)(implicit reader: ConfigReader[Config], ct: ClassTag[Config]): Config = {
+    getResultOrThrow[Config](loadConfig(path, namespace))
```
Comment (melrief) [inline core/src/main/scala/pureconfig/package.scala]: For consistency, add `[Config]` after `loadConfig` also here please

**Review by melrief — APPROVED** [review]

File: core/src/main/scala/pureconfig/package.scala (hunk 2 above)
Comment (ruippeixotog) [reply core/src/main/scala/pureconfig/package.scala]: Added in f3b55f6.

**Pull request merged**